src/components/ConditionalButtons/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+export default class ButtonsGroup extends React.Component {
```

Comment:
This is in a component called ConditionalButtons but exposes a class called ButtonsGroup.  Should it be that way?